src/components/disputeResolver/ManageDisputes/DisputesTable.tsx

```diff
@@ -175,7 +176,7 @@ export default function DisputesTable({ disputes }: Props) {
                     whiteSpace: "pre"
                   }}
                   onClick={() => {
-                    copyToClipboard(emailAddress).then(() => {
+                    copyToClipboard(sanitizeUrl(emailAddress)).then(() => {
```

Comment:
does this work or break the emailAddress that's copied even if it's a valid email address? @levalleux-ludo

Reply:
Checked.
Before:
![image](https://user-images.githubusercontent.com/7184124/233657140-d83acc40-f29d-44af-b311-38dc4213959b.png)

After:
![image](https://user-images.githubusercontent.com/7184124/233657180-a75ad0bf-6719-47a2-a621-1cfc5f42a4b0.png)